src/Stratis.Bitcoin.IntegrationTests/EnvironmentMockUpHelpers/NodeRunners.cs

```diff
@@ -0,0 +1,252 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using NBitcoin;
+using NBitcoin.Protocol;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Features.BlockStore;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.Features.MemoryPool;
+using Stratis.Bitcoin.Features.Miner;
+using Stratis.Bitcoin.Features.RPC;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.IntegrationTests
+{
+    public class BitcoinCoreRunner : INodeRunner
+    {
+        private string bitcoinD;
+
+        public BitcoinCoreRunner(string bitcoinD)
+        {
+            this.bitcoinD = bitcoinD;
+        }
+
+        private Process process;
+
+        public bool HasExited
+        {
+            get { return this.process == null && this.process.HasExited; }
+        }
+
+        public void Kill()
+        {
+            if (!this.HasExited)
+            {
+                this.process.Kill();
+                this.process.WaitForExit();
+            }
+        }
+
+        public void Start(string dataDir)
+        {
+            this.process = Process.Start(new FileInfo(this.bitcoinD).FullName,
+                "-conf=bitcoin.conf" + " -datadir=" + dataDir + " -debug=net");
```

Comment:
I think this string might be more readable using $"...{dataDir}..." syntax vs string concat?